dask/bytes/tests/test_local.py

```diff
@@ -256,7 +256,8 @@ def test_compressed_write(tmpdir):
 
 def test_open_files_write(tmpdir):
     tmpdir = str(tmpdir)
-    f = open_file_write([tmpdir+'/test1', tmpdir+'/test2'])
+    f = open_file_write([os.path.join(tmpdir, '/test1'),
+                         os.path.join(tmpdir, '/test2')])
```

Comment:
Sorry, you actually don't want a forward slash here at all.  You want `os.path.join(tmpdir, 'test1')`.  

If at some point you feel particularly masochistic you might install a Windows VM on your machine to run these tests locally.  We should also consider setting up Appveyor